eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/ChannelOpenSingleFunded.scala

```diff
@@ -292,13 +292,10 @@ trait ChannelOpenSingleFunded extends SingleFundingHandlers with ErrorHandlers {
               log.info(s"waiting for them to publish the funding tx for channelId=$channelId fundingTxid=${commitments.fundingTxId}")
               watchFundingTx(commitments)
               Funding.minDepthFundee(nodeParams.channelConf, commitments.localParams.initFeatures, fundingAmount) match {
-                case Some(fundingMinDepth) =>
-                  blockchain ! WatchFundingConfirmed(self, commitments.fundingTxId, fundingMinDepth)
-                  goto(WAIT_FOR_FUNDING_CONFIRMED) using DATA_WAIT_FOR_FUNDING_CONFIRMED(commitments, None, nodeParams.currentBlockHeight, None, Right(fundingSigned)) storing() sending fundingSigned
-                case None =>
-                  val (shortIds, channelReady) = acceptFundingTx(commitments, RealScidStatus.Unknown)
-                  goto(WAIT_FOR_CHANNEL_READY) using DATA_WAIT_FOR_CHANNEL_READY(commitments, shortIds, channelReady) storing() sending Seq(fundingSigned, channelReady)
+                case Some(fundingMinDepth) => blockchain ! WatchFundingConfirmed(self, commitments.fundingTxId, fundingMinDepth)
+                case None => blockchain ! WatchPublished(self, commitments.fundingTxId)
```

Comment:
The fundee case in single-funding wasn't problematic, I supposed you did it for consistency?

Reply:
Yes, I thought it was more consistent that way, especially in the tests.